src/framework/entity/entity_entity.js

```diff
@@ -82,38 +87,44 @@ pc.extend(pc, function () {
         if (system) {
             if (!this.c[type]) {
                 return system.addComponent(this, data);
-            } else {
+            }
```

Comment:
Checked in the fix for moveTowards when at target

Will fix the braces when I can figure out how to tell IntelliJ :)

> On 19 Jun 2015, at 10:23, vkalpias <notifications@github.com> wrote:
> 
> In src/framework/entity/entity_entity.js <https://github.com/playcanvas/engine/pull/326#discussion_r32813780>:
> 
> > @@ -82,38 +87,44 @@ pc.extend(pc, function () {
> >          if (system) {
> >              if (!this.c[type]) {
> >                  return system.addComponent(this, data);
> > -            } else {
> > +            }
> This is probably automatically changed by your text editor but we use brace and else in the same line :D
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/playcanvas/engine/pull/326/files#r32813780>.
> 


-- 
See our 2 minute video on DBS bringing gamification to Singapore here: 
www.3radical.com/resources/videos/#DBSVideo


3radical Limited, 1 Friary, Temple Quay, Bristol BS1 6EA, England.  
T: +44 117 344 5008 ¦ F : +44 117 344 5188

3radical Pte Ltd, 3 Church Street, Level 8 Samsung Hub, Singapore 049483.  
T: +65 6408 3711 ¦ F : +65 6408 0101

www.3radical.com

3radical Limited is a limited company registered in England and Wales 
under number 7872556.  The registered address of 3radical Limited is 1 
Friary, Temple Quay, Bristol BS1 6EA, England.

3radical Pte Ltd is a registered in Singapore under number 201312239K.  The 
registered address of 3radical Pte Ltd is 51 Goldhill Plaza, #07-10/11, 
Singapore 308900.

The information in this e-mail and any attachments may be confidential.  If 
you are not the intended recipient, please notify the sender as soon as 
possible and delete any copies. Unauthorised use, dissemination, 
distribution, publication or copying of this communication is prohibited.

Attachments to this email have been checked with virus detection 
software before transmission, however you should carry out virus checks 
before opening any attachment. 3radical Limited and 3radical Pte Ltd 
accepts no liability for any loss or damage which may be caused by software 
viruses.